site/content/en/docs/handbook/network_policy.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Network Policy"
+linkTitle: "Network Policy"
+weight: 10
+date: 2022-01-31
+description: >
+  Controlling traffic flowing through the cluster
+aliases:
+  - /docs/reference/network_policy
+---
+
+Minikube allows users to create and test network policies in the local Kubernetes cluster. This is useful since it allows the network policies to be, considered, built, and evaluated during the application development, as an integral part of the process rather than "bolted on" at the end of development.
```

Comment:
```suggestion
minikube allows users to create and test network policies in the local Kubernetes cluster. This is useful since it allows the network policies to be, considered, built, and evaluated during the application development, as an integral part of the process rather than "bolted on" at the end of development.
```